packages/contentstack-audit/src/modules/workflows.ts

```diff
@@ -0,0 +1,73 @@
+import { join, resolve } from 'path';
+import { existsSync, readFileSync } from 'fs';
+
+import { LogFn, ConfigType, ContentTypeStruct, CtConstructorParam, ModuleConstructorParam, Workflow } from '../types';
+
+import auditConfig from '../config';
+import { $t, auditMsg } from '../messages';
+import { values } from 'lodash';
+
+export default class Workflows {
+  public log: LogFn;
+  protected fix: boolean;
+  public fileName: any;
+  public config: ConfigType;
+  public folderPath: string;
+  public workflowSchema: Workflow[];
+  public ctSchema: ContentTypeStruct[];
+  public moduleName: keyof typeof auditConfig.moduleConfig;
+  public ctUidSet: Set<string>;
+  public fixWorkflowSchema: Workflow[];
+
+  constructor({ log, fix, config, moduleName, ctSchema }: ModuleConstructorParam & CtConstructorParam) {
+    this.log = log;
+    this.config = config;
+    this.fix = fix ?? false;
+    this.ctSchema = ctSchema;
+    this.workflowSchema = [];
+    this.moduleName = moduleName ?? 'workflows';
+    this.fileName = config.moduleConfig[this.moduleName].fileName;
+    this.folderPath = resolve(config.basePath, config.moduleConfig[this.moduleName].dirName);
+    this.ctUidSet = new Set(['$all']);
+    this.fixWorkflowSchema = [];
+  }
+
+  run() {
+    if (!existsSync(this.folderPath)) {
+      this.log(`Skipping ${this.moduleName} audit`, 'warn');
+      this.log($t(auditMsg.NOT_VALID_PATH, { path: this.folderPath }), { color: 'yellow' });
+    }
```

Comment:
If the folder does not exist it should not proceed to the next line we have to break or stop the process there itself.

Reply:
Got it I will add it

Reply:
added